psydac/feec/multipatch/operators.py

```diff
@@ -181,7 +181,24 @@ class ConformingProjection_V0( FemLinearOperator):
     #   - allow case without interfaces (single or multipatch)
     def __init__(self, V0h, domain_h, hom_bc=False, backend_language='python', storage_fn=None):
         """
-        :param load_dir: to allow to store/load the operator sparse matrix
+        Compute the V0 conforming projector
+
+        Parameters
+        ----------
+        V0h: <FemSpace>
+         The discrete space
+
+        domain_h: <Geometry>
+         The discrete domain of the projector
+
+        hom_bc : <bool>
+         Apply homogenous boundary conditions if True
+
+        backend_language: <str>
+         The backend used to accelerate the code
+
+        storage_fn:
+         filename to store/load the operator sparse matrix
         """
```

Comment:
Please move this docstring out of the `__init__`, as usual for a class.

By the way, you cannot use the words "Compute something" when documenting the class...

Reply:
it's not in the `__init__`  method , and there is no "Compute something". I think I have changed it in later commits

Reply:
OK